### DISCUSSION TITLE
Magic V.2

### DISCUSSION THREAD

**Discussion by littlesera** [littlesera/LSCG#299] [Ideas]
A place to collect the next big work related to the LSCG Magic system.
Core Improvements:
* [ ] Mana/Spell Slots, some sort of limiting, managable, and replenishable resource used to cast spells.
* [ ] Selective Dispelling
* [ ] Additional Spell Effects
* [ ] Additional spell-casting items
* [ ] Some sort of filtering search for spell list
* [ ] Magic items?

**Comment by jimmyG1** [comment]
win will magic be in the normal vershon it sounds like so mush fun

**Reply by littlesera** [reply]
Just released today! #301
